services/api-server/src/simcore_service_api_server/utils/solver_job_models_converters.py

```diff
@@ -252,19 +246,9 @@ def create_jobstatus_from_task(task: ComputationTaskGet) -> JobStatus:
         job_id=task.id,
         state=task.state,
         progress=PercentageInt((task.pipeline_details.progress or 0) * 100.0),
-        submitted_at=datetime.utcnow(),
+        submitted_at=task.submitted or arrow.utcnow().datetime,
```

Comment:
Q: why would this happen? shouldn't ``task.submitted`` be not `None` when the task is submitted? 
Moreover, can it happen that either `started_at` or `stopped_at` < `submitted_at` if `submitted_at=arrow.utcnow().datetime` ?

Reply:
submitted can be none if there are no computational services in the pipeline.